src/index.html

```diff
@@ -16,17 +19,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      class="search-bar search-bar-big"
+      data-qa="big"
+    >
+      <label class="search-bar__label search-bar__label-big">
+        <input
+          data-qa="keypress"
+          class="search-bar__input search-bar__input-big"
+          type="text"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
The attributes for the input tag are correctly formatted since there are more than two attributes. However, ensure that the line length does not exceed 80 characters.